### PR TITLE
Add visibility flag to kubectl gs template catalog

### PR DESCRIPTION
**Pull request by rossf7**
Follow up to https://github.com/giantswarm/giantswarm/issues/19941

Code: https://github.com/giantswarm/kubectl-gs/pull/710

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/app-platform/create-catalog/index.md [hunk 1]
```diff
@@ -117,7 +118,8 @@ kubectl gs template catalog \
   --namespace org-example \
   --description "An example Catalog" \
   --url https://example.github.io/example-catalog/ \
-  --logo https://example.com/logos/example-logo.png
+  --logo https://example.com/logos/example-logo.png \
+  --visibility public
```
Comment (MarcelMue) [inline src/content/app-platform/create-catalog/index.md:122]: Does it make sense to add this? As it's the default

Comment (rossf7) [inline src/content/app-platform/create-catalog/index.md:122]:
```suggestion
  --logo https://example.com/logos/example-logo.png
```

Comment (rossf7) [reply src/content/app-platform/create-catalog/index.md]: @MarcelMue Damn I overthought this. My idea was to show you can change it but its already covered in the text.

I've removed. PTAL

**Review by MarcelMue — APPROVED** [review]

**Comment by marians** [comment]
As this feature has been released already, can we publish this?

**Comment by rossf7** [comment]
@marians Yes, it's merged now.

**Pull request merged**